.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,29 @@
+name: close-stale
+
+on:
+  schedule:
+  - cron: '0 2 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  close-stale:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Close Stale Issues
+      uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has been automatically closed due to inactivity for more than 90 days. Please feel free to reopen if you still have this issue.'
+        stale-pr-message: 'This pull request has been automatically closed due to inactivity for more than 90 days. Please feel free to reopen if you still have this pull request.'
```

Comment:
`stale-issue-message` and `stale-pr-message` are written in a way which implies that being marked as stale leads to instant closure. Should these read something like below instead?

This [issue/pull request] has been inactive for more than 90 days. If there is no further activity, it will be automatically closed in seven days time. You can reopen the [issue/pull request] if it is still relevant.

Reply:
Addressed in the latest commit (https://github.com/forrtproject/forrtproject.github.io/pull/145/commits/1781234cbac2e8155988df92ca4b676e65b146a0)

* Reduced the stale counter to 60 days
* Added `close-[pr/issue]-message` options.